### PR TITLE
add promoteSubMenuItemToMainMenu omt menu snipper

### PR DESCRIPTION
**Pull request by DBuit**

### CODE REVIEW

**Comment by DBuit** [comment]
@emielb Zou je naar dit pull request kunnen kijken? Is maar een kleine wijziging maar is al in omt doorgevoerd zou mooi zijn als hij hier ook in zit.

File: syntaxes/omt.tmLanguage.json [hunk 1]
```diff
@@ -31,6 +31,13 @@
                         "3": { "patterns": [{ "include": "#_comments-only" }] }
                     }
                 },
+                {
+                    "match": "(?:\\G|^) +(promoteSubMenuItemToMainMenu): *(\\w*)(.*)$",
```
Comment (emielb) [inline syntaxes/omt.tmLanguage.json]: The capturing for the last group needs to be added, to enable highlighting just comments at the end of the line.

**Comment by emielb** [comment]
After adding the comment-only highlighting, please also add the changes in this PR to the changelog and update the version to 2.1.1 :)

File: syntaxes/omt.tmLanguage.json (hunk 1 above)
Comment (DBuit) [reply syntaxes/omt.tmLanguage.json]: @emielb i think i added the write part with comments only, let me know if it's ok now

Comment (emielb) [reply syntaxes/omt.tmLanguage.json]: 👍

**Comment by DBuit** [comment]
@emielb I can't add any additional reviewers to the pull request. As i need atleast 2 approved reviews, could you add more reviewers?

**Comment by emielb** [comment]
> @emielb I can't add any additional reviewers to the pull request. As i need atleast 2 approved reviews, could you add more reviewers?

**Pull request merged**